components/CharacterCard/CharacterCard.js

```diff
@@ -1 +1,31 @@
-export function CharacterCard() {}
+
+
+
+
+export function CharacterCard() {
```

Comment:
Where do all the variables come from ( picSource, charName,...). I think you need some parameters here.